.gitignore

```diff
@@ -112,6 +112,8 @@ deployment/ansible/*.retry
 *.aux.xml
 
 /data/
-/notebooks/
 /.vscode/
 /.history/
+
+/notebooks/*
```

Comment:
There are some places in the notebooks where you display an image with matplotlib and I think it would look better if you used `axes('off')`

Reply:
I left those axes on to call attention to the dimensions of the rasters being plotted.